mkContainer/mkContainer.nix

```diff
@@ -96,6 +95,41 @@ let
     ];
   };
 
+  # For field definitions, see `ActivateCtx` in `flox-core`
+  activateCtx = {
+    mode = "${activationMode}";
+    shell = {
+      Bash = "${containerPkgs.bashInteractive}/bin/bash";
+    };
+    env = "${environment}";
+    run_args = [ ];
+    invocation_type = null;
+    remove_after_reading = false;
+    env_description = "${containerName}";
+    env_cache = "/tmp";
+    flox_env_log_dir = "/log/flox";
+    flox_runtime_dir = "/run/flox";
+    prompt_color_1 = "99";
+    prompt_color_2 = "141";
+    flox_activate_store_path = "${environment}";
+    interpreter_path = "${environment}";
+    flox_prompt_environments = "floxenv";
+    set_prompt = true;
+    flox_activate_start_services = false;
+    flox_services_socket = "/run/flox/services.sock";
```

Comment:
suggestion nonblocking: that's unused since flox_activate_start_services = false, right? So I think we can make it null?